build.gradle

```diff
@@ -24,29 +24,242 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+plugins {
+    id 'idea'
+    id 'eclipse'
+    id 'java'
+    id 'net.saliman.cobertura' version '3.0.0'
+    id 'distribution'
+    id 'signing'
+    id 'maven'
+    id 'org.ajoberstar.git-publish' version '3.0.0'
+    id 'nebula.release' version '15.2.0'
+}
+
+group 'com.emc.ecs'
 description = 'Smart REST Client - JAX-RS (Jersey) REST client that provides client-side load balancing with a pluggable host list provider'
 
+// name of the github project repository
 ext.githubProjectName = 'smart-client-java'
+// URL to github project
+ext.githubProjectUrl = "https://github.com/twincitiesguy/${githubProjectName}"
+// git remote scm address
+ext.githubScmUrl = "scm:git@github.com:twincitiesguy/${githubProjectName}.git"
+// git remote https address
+ext.githubRemoteUrl = "https://github.com/twincitiesguy/${githubProjectName}.git"
+// license info
+ext.licenseName = 'The BSD 3-Clause License'
+ext.licenseUrl = 'http://opensource.org/licenses/BSD-3-Clause'
 
-buildscript {
-    ext.commonBuildVersion = '1.6'
-    ext.commonBuildDir = "https://raw.githubusercontent.com/emcvipr/ecs-common-build/v$commonBuildVersion"
-    apply from: "$commonBuildDir/ecs-publish.buildscript.gradle", to: buildscript
-}
+defaultTasks 'distZip'
 
-apply from: "$commonBuildDir/ecs-publish.gradle"
+repositories {
+    mavenCentral()
+    mavenLocal()
+    maven {
+        url "https://maven.atlassian.com/3rdparty/"
+    }
+}
 
 dependencies {
-    compile 'com.sun.jersey:jersey-client:1.19.4',
-            'com.sun.jersey.contribs:jersey-apache-client4:1.19.4',
-            'org.codehaus.jackson:jackson-jaxrs:1.9.13',
-            'org.codehaus.jackson:jackson-xc:1.9.13',
+    implementation 'log4j:log4j:1.2.17',
```

Comment:
remove log4j from compile dependencies - we use slf4j so users can choose their own logger

Reply:
done